os/kernel/binary_manager/Kconfig

```diff
@@ -45,6 +45,14 @@ config BINMGR_UPDATE
 	---help---
 		Enables Binary Manager Update APIs.
 
+config BINMGR_ALLOW_SAME_VERSION_UPDATE
```

Comment:
This is the config of binary update. Let's follow the same naming policy.
BINMGR_UPDATE_SAME_VERSION